src/daycare.c

```diff
@@ -32,6 +32,8 @@
 
 // Combination of RSE's Day-Care (re-used on Four Island), FRLG's Day-Care, and egg_hatch.c
 
+extern struct Evolution gEvolutionTable[][EVOS_PER_MON];
```

Comment:
Why is const removed here?

Reply:
For consistency. I can see if it matches with const, I didn't think it really mattered.